CMakeLists.txt

```diff
@@ -219,8 +233,13 @@ else()
         message(WARNING "mpi.h not found!")
     endif()
     ########################## End of fix for OpenMPI ############################################
+
+    # Reactivate previous CMAKE_REQUIRED_FLAGS  
+    set( CMAKE_REQUIRED_FLAGS "${T8CODE_OLD_CMAKE_REQUIRED_FLAGS}" )
+    unset( T8CODE_OLD_CMAKE_REQUIRED_FLAGS )
```

Comment:
```suggestion
```
this should not be necessary, because the else from line 171 opens its own scope